docs/runs.md

```diff
@@ -64,6 +66,19 @@ A `Run` definition supports the following fields:
 
 ### Specifying the target Custom Task
 
+A custom task resource's `Spec` may be directly embedded in the `Run` or it may
+be referred to by a `Ref`. But, not both at the same time.
+
+1. [Specifying the target Custom Task with ref](#specifying-the-target-custom-task-with-ref)
+   Referring a custom task (i.e. `Ref` ) promotes reuse of custom task definitions.
+
+2. [Specifying the target Custom Task by embedding its spec](#specifying-the-target-custom-task-by-embedding-its-spec)
```

Comment:
ditto 🙃 valid reference here is ` #2-specifying-the-target-custom-task-by-embedding-its-spec`